src/tests/reducers/header.reducer.test.js

```diff
@@ -0,0 +1,32 @@
+import {
+  headerReducer,
+  DEFAULT_STATE
+} from "../../redux/reducers/header.reducer"
+import {
+  HEADER_ACTIONS,
```

Comment:
@jprittie @astr0-4 I see this warning too - if we're not using HEADER_ACTIONS let's delete it and we're good to go!